mini-graph-lib.js

```diff
@@ -8,10 +8,11 @@ function getValueArray(items) {
   return items.map(item => Number(item.state) || 0);
```

Comment:
There is already a check for unknown/unavailable states that should work in the current implementation, however your solution is superior.
It's a much better idea filtering out unknown states instead of just assign them 0 values.

Reply:
But we should remove `|| 0` here, for your implementation to work properly.